xercesc.sh

```diff
@@ -0,0 +1,33 @@
+package: XercesC
+version: v3.1.4
+source: https://github.com/PMunkes/XercesC
+tag: v3.1.4
+env:
+  XERCESC_INST_DIR: "$XERCESCINST"
```

Comment:
Ok, great. I think this is the only controversial point, then. I would still like to give it a try myself, but I am not sure I can do it before the 25th of July. Are you blocked by this? If not, would you mind waiting?